index.ftd

```diff
@@ -20,6 +20,11 @@ github-url: https://github.com/fastn-community/set-2-ds/
 github-icon: true
 sidebar: true
 
+
+-- page.body:
+
+Hello
+
```

Comment:
@MeenuKumari28 plz add the body as given here: https://fastn-community.github.io/set-2-ds/